components/AdminTable/AdminContactPointsTable/AdminContactPointsTable.vue

```diff
@@ -0,0 +1,189 @@
+<template>
+  <AdminTable>
+    <thead>
+      <AdminTableTh scope="col">
+        {{ t("Contact point name") }}
+      </AdminTableTh>
+      <AdminTableTh class="w-44">
+        {{ t("Role") }}
+      </AdminTableTh>
+      <AdminTableTh>
+        {{ t("Contact Email") }}
+      </AdminTableTh>
+      <AdminTableTh
+        scope="col"
+      >
+        {{ t("Contact Url") }}
+      </AdminTableTh>
+      <AdminTableTh
+        class="w-32"
+        scope="col"
+      >
+        {{ t("Action") }}
+      </AdminTableTh>
+    </thead>
+    <tbody>
+      <tr
+        v-for="contactPoint in contactPoints"
+        :key="contactPoint.id"
+      >
+        <td class="font-bold">
+          {{ contactPoint.name }}
+        </td>
+        <td>
+          <AdminBadge
+            size="xs"
+            type="secondary"
+          >
+            {{ getRoleLabel(contactPoint) }}
+          </AdminBadge>
+        </td>
+        <td>
+          {{ contactPoint.email }}
+        </td>
+        <td>
+          <div class="break-all truncate">
+            {{ contactPoint.contact_form }}
+          </div>
+        </td>
+        <td>
+          <ModalWithButton
+            :title="$t('Edit the contact point')"
+            size="lg"
+            @open="newContactForm = contactPoint"
+            @close="newContactForm = null"
+          >
+            <template #button="{ attrs, listeners }">
+              <BrandedButton
+                color="secondary-softer"
+                :icon="RiPencilLine"
+                icon-only
+                size="xs"
+                keep-margins-even-without-borders
+                v-bind="attrs"
+                v-on="listeners"
+              >
+                {{ $t('Transfer') }}
```

Comment:
I think there is a wording issue here but it doesn't seem to be taken into account though. Should it be a `title` props on BrandedButton?

Reply:
I'm adding a title too for sighted user